app/assets/javascripts/darkswarm/controllers/checkout/checkout_controller.js.coffee

```diff
@@ -7,6 +7,7 @@ Darkswarm.controller "CheckoutCtrl", ($scope, localStorageService, Checkout, Cur
   prefix = "order_#{Checkout.order.id}#{CurrentUser.id or ""}#{CurrentHub.hub.id}"
 
   for field in $scope.fieldsToBind
+    # SHIPPING_METHOD
```

Comment:
All these tags from the git history once done. I will help us identify the offenses to fix.